src/argonms/common/loading/item/KvjItemDataLoader.java

```diff
@@ -270,13 +267,6 @@ private void processReqStat(LittleEndianReader reader, Integer oId) {
 		reqStats.get(oId)[stat] = value;
 	}
 
-	private List<Integer> processScrollReqs(LittleEndianReader reader) {
```

Comment:
It'd be nice to stage this in a separate commit instead of bundling it together with the rest of the changes.

Reply:
This will be re-implemented later on in the PR because I actually found the NXNode for it.
Still will be unused and might be implemented later as a functionality when I know more about it.

Reply:
Re-implemented https://github.com/geospiza-fortis/argonms-server/pull/15/commits/41c0f1e78d94ab357fc6f79ce1517a23a3337654
Done https://github.com/geospiza-fortis/argonms-server/pull/15/commits/eda49f8ca820bf9bbfadac21c75cdcc82ee73eeb